Modules/Devices/src/Devices.Domain/Entities/Identities/Identity.cs

```diff
@@ -330,7 +330,8 @@ public void ChangeFeatureFlags(Dictionary<FeatureFlagName, bool> featureFlags)
         {
             FeatureFlags.Set(keyValuePair.Key, keyValuePair.Value);
         }
-        RaiseDomainEvent(new FeatureFlagsOfIdentityChangedDomainEvent(this));
+
+        RaiseDomainEvent(new FeatureFlagsOfIdentityChangedDomainEvent(Address.Value));
```

Comment:
We usually pass the whole domain object so that the event's constructor can do the mapping (just as we did it before your changes). Why did you decide to go a different way?